packages/gafl-webapp-service/src/pages/renewals/identify/route.js

```diff
@@ -6,40 +6,33 @@ import Boom from '@hapi/boom'
 
 const getData = async request => {
   // If we are supplied a permission number, validate it or throw 400
-  const referenceNumber = request.params.referenceNumber
-  const validatePermissionNumber = validation.permission.permissionNumberUniqueComponentValidator(Joi).validate(referenceNumber)
-
-  if (validatePermissionNumber.error) {
-    throw Boom.forbidden('Attempt to access the authentication page with an invalid permission number')
-  }
-
-  // Save it in the cache
-  await request.cache().helpers.status.setCurrentPermission({ referenceNumber })
-
-  // Get the authentication status
-  const { authentication } = await request.cache().helpers.status.getCurrentPermission()
-
-  /*
-   * If the authorization is denied then write a page-error object
-   */
-  if (authentication && !authentication.authorized) {
-    return { referenceNumber, error: { referenceNumber: 'string.invalid' } }
+  const permission = await request.cache().helpers.status.getCurrentPermission()
+
+  if (permission.referenceNumber) {
+    const validatePermissionNumber = validation.permission
+      .permissionNumberUniqueComponentValidator(Joi)
+      .validate(permission.referenceNumber)
+    if (validatePermissionNumber.error) {
```

Comment:
I'm not convinced that we should be validating the permission number in the frontend.  If I change the permission number validator to disallow vowels (due to the risk of generating a number with expletives) then this will prevent existing users from renewing (current licence numbers allow an E).  This also gets messy from a change management perspective if future changes are made.